Default (Windows).sublime-keymap

```diff
@@ -1,13 +1,13 @@
 [
   // Find the dependents for the current module
   {
-    "keys": ["control+alt+up"],
+    "keys": ["ctrl+shift+up"],
```

Comment:
I really wanted uniformity about the key bindings, but as an OSX user, I'll defer to your judgement on more ergonomic key bindings.